### DISCUSSION TITLE
compile issue

### DISCUSSION THREAD

**Discussion by yoonk0511** [MDIL-SNU/SIMPLE-NN_v2#95] [Q&A]
Hi,
I copied pair_nn * to /LAMMPS/src, 
and compiled with make mpi command.
But There are some issues

"../pair_nn_replica.cpp: In member function ‘virtual void LAMMPS_NS::PairREPLICA::init_style()’:
../pair_nn_replica.cpp:850:35: error: ‘int LAMMPS_NS::NeighRequest::half’ is protected within this context
  850 |     neighbor->requests[irequest]->half = 0;
      |                                   ^~~~
In file included from ../pair_nn_replica.cpp:30:
../neigh_request.h:55:7: note: declared protected here
   55 |   int half;    // half neigh list (set by default)
      |       ^~~~
../pair_nn_replica.cpp:851:35: error: ‘int LAMMPS_NS::NeighRequest::full’ is protected within this context
  851 |     neighbor->requests[irequest]->full = 1;
      |                                   ^~~~
In file included from ../pair_nn_replica.cpp:30:
../neigh_request.h:56:7: note: declared protected here
   56 |   int full;    // full neigh list
      |       ^~~~ "

How can I compile LAMMPS including simple-nn

**Comment by JisuJung928** [comment]
Hi, yoonk0511

I apologize for being too late. Did you solve the problem?
I am sure that you can compile the latest version without the error.
Please let me know if you still are struggling with the problem.

Best regard,
Jisu.
